WooCommerce/UITestsFoundation/XCTest+Extensions.swift

```diff
@@ -114,8 +114,44 @@ extension XCUIElement {
         startCoordinate.press(forDuration: 0.01, thenDragTo: destination)
     }
 
-    public func getTextVisibilityCount(text: String) throws -> Int {
-        let predicate = NSPredicate(format: "label CONTAINS[c] %@", text)
+    func getStaticTextVisibilityCount(textToFind: String) throws -> Int {
+        let predicate = NSPredicate(format: "label CONTAINS[c] %@", textToFind)
         return XCUIApplication().staticTexts.containing(predicate).count
     }
+
+    public func assertTextVisibilityCount(textToFind: String) {
+        let count = try! getStaticTextVisibilityCount(textToFind: textToFind)
+        XCTAssertTrue(count == 1, "Expected '\(textToFind)' to appear once, but it appeared '\(count)' times!")
+    }
+
+    // Parent is using accessibilityIdentifier, child is using staticText
+    func verifyChildElementOnParentCell(parent: String, child: String) throws -> Bool {
+        let parentPredicate = NSPredicate(format: "identifier == %@", parent)
+        let childPredicate = NSPredicate(format: "label ==[c] %@", child)
+
+        return XCUIApplication().tables.cells.matching(parentPredicate).children(matching: .staticText).element(matching: childPredicate).firstMatch.exists
+    }
+
+    public func assertElementExistsOnCell(mainCell: String, elementToFind: String) {
```

Comment:
Naming nitpick:

```suggestion
    public func assertElement(matching: String, existsOnCellWithIdentifier cellIdentifier: String) {
```

The [Swift API Guidelines](https://swift.org/documentation/api-design-guidelines/) have good principles to follow to pick names for Swift code, and decent examples to get an idea.

The way I have internalized those, which may or may not be correct, is that we want code that reads well.

IMHO, "assert element matching `foo` exists on cell with identifier `bar`" reads like an English sentence.